index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
its good practice to change the title of your site

Reply:
I think it would be better to say why it's a good practice. Giving the "why" when asking for changes helps with more effective communication.